Mathlib/Algebra/Group/Int.lean

```diff
@@ -183,6 +186,92 @@ lemma eq_one_or_neg_one_of_mul_eq_neg_one (h : u * v = -1) : u = 1 ∨ u = -1 :=
   Or.elim (eq_one_or_neg_one_of_mul_eq_neg_one' h) (fun H => Or.inl H.1) fun H => Or.inr H.1
 #align int.eq_one_or_neg_one_of_mul_eq_neg_one Int.eq_one_or_neg_one_of_mul_eq_neg_one
 
+/-! #### Parity -/
```

Comment:
This would make it a subsection of Units, which I don't think is intended.

```suggestion
/-! ### Parity -/
```

Reply:
Ah I meant for "Units" to be a subsection of "Miscellaneous lemmas".